### ISSUE TITLE
No mention of $createElement in the API?

### ISSUE THREAD

**Issue by sqal** [vuejs/vuejs.org#658]
I don't see anywhere `vm.$createElement` being mentioned in the docs   It's not a private method, right? judging from https://github.com/vuejs/vue/blob/f1c38674f28116e1e335873ecbc6716fb70448a3/src/core/instance/render.js#L39 and i actually find it useful in some cases.

**Comment by LinusBorg** [comment]
Hm ... It's not a private function really, no.

It probably slipped through because the only real use for it is in the `render` function, to which it is passed as the argument anyway:

https://vuejs.org/v2/api/#render

But it might be worth documenting.

**Comment by sqal** [comment]
> the only real use for it is in the render function, to which it is passed as the argument anyway:

Agree, but sometimes `render` logic can be very complex so it's useful to move some of the code to the custom method function. Then you have 2 choices, a) pass the `createElement` from the `render` down to the method or b) just use `this.$createElement`

**Comment by AlexandreBonaventure** [comment]
Some third party plugins are using this API : [see element-ui]( http://element.eleme.io/#/en-US/component/message-box#customization)

**Comment by LinusBorg** [comment]
Every component with a custom render function uses this API.

**Comment by lzhoucs** [comment]
It seems vuetify is also one of them: https://github.com/vuetifyjs/vuetify/blob/dev/src/components/VDataTable/mixins/body.js#L9

**Comment by dohomi** [comment]
It would be great to document it - just found out about it on accident. Very useful helper for JS based components

**Comment by DominusVilicus** [comment]
Why hasn't this been documented yet?

**Comment by MrJmpl3** [comment]
I searching why Vuetify use "this.$createElement" without jsx babel, and now finding the why!
 
The documentation is missing this attribute. u.u
